common/src/main/java/whocraft/tardis_refined/common/tardis/control/Control.java

```diff
@@ -65,4 +77,27 @@ public boolean canUseControl(TardisLevelOperator tardisLevelOperator, Control co
         return !isDeskopWaiting && TardisEvents.PLAYER_CONTROL_INTERACT.invoker().canControlBeUsed(tardisLevelOperator, control, controlEntity) == EventResult.pass();
     }
 
+    public ResourceLocation getId(){
+        return this.id;
+    }
+    public String getTranslationKey() {return this.langId;}
+    public String getSerializedName() {
+        // todo Temporary - is bad :(
+        return this.getId().getPath();
+    }
```

Comment:
getSerialializedName looks to be a holdover from when the Control was an Enum, if this is not being used we don't need this.